src/neo/Cryptography/Crypto.cs

```diff
@@ -3,21 +3,41 @@
 
 namespace Neo.Cryptography
 {
+    /// <summary>
+    /// A cryptographic helper class.
+    /// </summary>
     public static class Crypto
     {
+        /// <summary>
+        /// Gets a 160-bit hash value of a specified message.
+        /// </summary>
+        /// <param name="message">The message to be hashed.</param>
+        /// <returns>160-bit hash value.</returns>
```

Comment:
It's `byte[]`, so it'd be nice to mention that this method returns exactly 20 bytes.